containers/package-builder/checkout-golem-version.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
```

Comment:
You could have a single script that works for both golem and concent if you added more parameters.

Reply:
Done.